src/pages/dashboard/index.tsx

```diff
@@ -0,0 +1,235 @@
+                    <div className="ml-5 w-0 flex-1">
+                      <dl>
+                        <dt>
+                          <div className="text-cool-gray-500 truncate text-sm font-medium leading-5">
+                            {t("Notebooks")}
+                          </div>
+                        </dt>
+                        <dd>
+                          <div className="text-cool-gray-900 text-lg font-medium leading-7">
+                            {data?.totalNotebooks}
+                          </div>
+                        </dd>
+                      </dl>
+                    </div>
+                  </div>
+                </div>
+                <div className="bg-cool-gray-50 px-5 py-3">
+                  <div className="text-sm leading-5">
+                    <Link
+                      href="/notebooks"
+                      className="font-medium text-blue-600 transition duration-150 ease-in-out hover:text-blue-900"
+                    >
+                      {t("View all")}
+                    </Link>
+                  </div>
+                </div>
+              </Block>
+              <Block>
+                <div className="p-5">
+                  <div className="flex items-center">
+                    <div className="flex-shrink-0">
+                      <BeakerIcon className="text-cool-gray-400 h-6 w-6" />
+                    </div>
+                    <div className="ml-5 w-0 flex-1">
+                      <dl>
+                        <dt>
+                          <div className="text-cool-gray-500 truncate text-sm font-medium leading-5">
+                            {t("Pipelines")}
+                          </div>
+                        </dt>
+                        <dd>
+                          <div className="text-cool-gray-900 text-lg font-medium leading-7">
+                            {data?.dags.totalItems}
+                          </div>
+                        </dd>
+                      </dl>
+                    </div>
+                  </div>
+                </div>
+                <div className="bg-cool-gray-50 px-5 py-3">
+                  <div className="text-sm leading-5">
+                    <Link
+                      href="/pipelines"
+                      className="font-medium text-blue-600 transition duration-150 ease-in-out hover:text-blue-900"
+                    >
+                      {t("View all")}
+                    </Link>
+                  </div>
+                </div>
+              </Block>
+            </div>
+          </div>
+
+          <div className="my-12">
+            <Title
+              level={4}
+              className="text-lg font-medium leading-6 text-gray-700"
+            >
+              {t("Last Activity")}
+            </Title>
+            <div className="mt-4">
+              <Block>
+                <DataGrid data={data?.lastActivities || []}>
+                  <TextColumn
```

Comment:
This field contains html that needs to be evaluated.
<img width="316" alt="CleanShot 2022-11-10 at 10 41 19@2x" src="https://user-images.githubusercontent.com/1607549/201055847-d50734ee-e165-4221-8817-17144ba27ddf.png">

Reply:
For this can I use html-react-parser. What's your thought?

Reply:
No need, you can just use the basic attribute provided by react

Reply:
You can use `dangerouslySetInnerHTML` since the html comes from the backend

Reply:
Okay. Thank